contracts/lz/SimpleLzApp.sol

```diff
@@ -0,0 +1,111 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.10;
+
+import "@rari-capital/solmate/src/auth/Owned.sol";
+import "./interfaces/ILayerZeroUserApplicationConfig.sol";
+import "./interfaces/ILayerZeroEndpoint.sol";
+
+/**
+ * @title SimpleLzApp
+ * @notice LayerZero-enabled contract that has only one trusted remote chain and sends messages cross-chain (does NOT
+ * receive) to remote contracts provided by the concrete class.
+ * NOTE: this contract is ownable only to set layerzero configs.
+ */
+abstract contract SimpleLzApp is Owned, ILayerZeroUserApplicationConfig {
```

Comment:
Please, instead of creating a new file, let's import `SimpleLzApp` adding LayerZero repo as dependency